acis_thermal_check/main.py

```diff
@@ -141,11 +141,12 @@ def run(self, args, override_limits=None):
             print(f"acis_thermal_check version {version}")
             return
 
-        # First, record the selected state builder in the class attributes
-        self.state_builder = make_state_builder(args.state_builder, args)
-
+        # First, do some initial setup and log important information.
         proc = self._setup_proc_and_logger(args)
 
+        # Record the selected state builder in the class attributes
```

Comment:
Did it not work if you didn't move this after `_setup_proc_and_logger`?

Reply:
I moved it because I wanted to be sure that the output directory was created before Backstop History was instantiated.,

Reply:
Ah, good